src/views/ipv/pyi-new-details.njk

```diff
@@ -4,6 +4,11 @@
 {% set pageTitleName = 'pages.pyiNewDetails.title' | translate %}
 {% set googleTagManagerPageId = "pyiNewDetails" %}
 
+{% set errorState = pageErrorState %}
```

Comment:
You'll also need to refactor the way we set the form in the template which has changed, you can see via the other pages currently in there